lib/controller.dart

```diff
@@ -281,9 +281,13 @@ void _computeChartBuckets(List<model.Chart> charts) {
   if (_dataNormalisationEnabled) {
     for (var chart in charts) {
       for (var chartCol in chart.fields) {
+        var filterValuesPercent = chartCol.bucket[0] * 100 / _filterValuesCount;
+        var comparisonFilterValuesPercent =
+            chartCol.bucket[1] * 100 / _comparisonFilterValuesCount;
+
         chartCol.bucket = [
-          (chartCol.bucket[0] * 100 / _filterValuesCount).truncate(),
-          (chartCol.bucket[1] * 100 / _comparisonFilterValuesCount).truncate()
+          num.parse(filterValuesPercent.toStringAsFixed(2)),
```

Comment:
consider moving this type of processing to a helper function (or if you're feeling brave, you could create an extension for `num`s)

Also, turning it into a string and then parsing it again seems like a bit of a hacky way to ensure you get a value with two significant decimals... and a bit slow as well.

I would be inclined to suggest using just maths operations instead, something like `(filterValuesPercent * 100).truncate() / 100` should do the trick.

However, if the only reason for doing this is for presentation purposes (ie, so that the numbers look good in a chart), you could just leave the `num`s here with whatever precision they have been computed, and configure the charting library to show them with the precision you want.

Reply:
Ah nice - I added extension for `num`. Noted about the num to string and back being slow - used the math operation but with round instead of truncate